src/ert/gui/tools/plot/plot_ensemble_selection_widget.py

```diff
@@ -51,7 +61,8 @@ def __init__(self, ensembles: List[EnsembleObject]):
             self.addItem(it)
             self._ensemble_count += 1
 
-        self.viewport().setMouseTracking(True)
+        if (viewport := self.viewport()) is not None:
+            viewport.setMouseTracking(True)
```

Comment:
Why the walrus operator here? We don't need `viewport` later on?

```
if self.viewport():
    self.viewport().setMouseTracking(True)
```

Reply:
Because two calls to self.viewport() could first return `None` and then something else. We kind of know this is a getter, but `mypy` doesn't care.